compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/methodgen/MethodGen.java

```diff
@@ -946,6 +946,17 @@ private void createLocalVariableTable(BIRFunction func, BIRVarToJVMIndexMap inde
                 if (localVar.endBB != null) {
                     endLabel = labelGen.getLabel(funcName + endBB.id.value + "beforeTerm");
                 }
+            } else if (localVar.kind == VarKind.LET) {
+                if (localVar.startBB != null) {
+                    startLabel = labelGen.getLabel(funcName + SCOPE_PREFIX + localVar.insScope.id);
+                }
+                for (int j = i + 1; j < func.localVars.size(); j++) {
+                    BIRVariableDcl syntheticVar = func.localVars.get(j);
+                    if (syntheticVar.kind == VarKind.SYNTHETIC && !isCompilerAddedVars(syntheticVar.metaVarName)) {
```

Comment:
What happen if there are not variables after let expression variable. 
public function main() {
    string greeting = let string hello = "Hello ", string ballerina = "Ballerina!" 
                        in hello + ballerina;
}